op-e2e/actions/reorg_test.go

```diff
@@ -148,11 +148,11 @@ func ReorgFlipFlop(gt *testing.T, deltaTimeOffset *hexutil.Uint64) {
 	verifEngClient := verifierEng.EngineClient(t, sd.RollupCfg)
 	checkVerifEngine := func() {
 		// TODO: geth preserves L2 chain with origin A1 after flip-flopping to B?
-		//ref, err := verifEngClient.L2BlockRefByLabel(t.Ctx(), eth.Unsafe)
-		//require.NoError(t, err)
-		//t.Logf("l2 unsafe head %s with origin %s", ref, ref.L1Origin)
-		//require.NotEqual(t, verifier.L2Unsafe().Hash, ref.ParentHash, "TODO off by one, engine syncs A0 after reorging back from B, while rollup node only inserts up to A0 (excl.)")
-		//require.Equal(t, verifier.L2Unsafe(), ref, "verifier safe head of engine matches rollup client")
+		// ref, err := verifEngClient.L2BlockRefByLabel(t.Ctx(), eth.Unsafe)
+		// require.NoError(t, err)
```

Comment:
This is not part of this PR but confused as to why this code is commented out in the first place

Reply:
My auto-save-formatter created this diff. No idea what's going on here. git blame shows that @protolambda added this TODO on 2022-10-22, so it's quite dated. Would prefer not to deal with this in this PR. I'll check with him if this can be deleted.

Reply:
Discussed internally with proto that we can delete this TODO code, the test has the right assertions and good coverage.